samples/Samples.Mvc5/Controllers/HomeController.cs

```diff
@@ -131,6 +133,35 @@ public ActionResult About()
         /// <returns>The <see cref="ActionResult"/>.</returns>
         public ActionResult ResultsAuthorization() => View();
 
+        public async Task<ActionResult> CallAsync()
+        {
+            var profiler = MiniProfiler.Current;
+            var service = new FooService();
+            using (profiler.Step("action"))
+            {
+                var tasks = new List<Task>();
+                for (int i = 0; i < 10; i++)
+                {
+                    tasks.Add(service.FooAsync());
+                }
+                await Task.WhenAll(tasks);
+                return Content("All good");
+            }
+        }
+
+        class FooService
+        {
+            public async Task FooAsync()
+            {
+                var profiler = MiniProfiler.Current;
+                await Task.Delay(10).ConfigureAwait(false);
```

Comment:
continuations of this await will execute in parallel since await is using ConfigureAwait(false) and the calling method is using Task.WhenAll instead of awaiting individual executions.